src/main/java/ua/danit/final_project/entities/User.java

```diff
@@ -0,0 +1,43 @@
+package ua.danit.final_project.entities;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import java.io.Serializable;
+import java.util.List;
+
+@Entity
+@Table(name = "user")
+@Data
+@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
+public class User implements Serializable {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+
+  @Column(name = "u_login", unique = true, nullable = false)
+  private String login;
+
+  @JsonIgnore
+  @Column(name = "u_password", nullable = false)
+  private String password;
+
+  @ManyToMany(cascade = CascadeType.ALL, fetch = FetchType.LAZY)
+  @JoinTable(name = "user_role",
+          joinColumns = {@JoinColumn(name = "u_id")},
+          inverseJoinColumns = {@JoinColumn(name = "r_id")})
+  public List<Role> roles;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make "roles" private or transient. ([squid:S1948](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1948))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=80&issues=AWVMbp7plRPS_ea-WxrT&open=AWVMbp7plRPS_ea-WxrT)

[ISSUE_KEY]: AWVMbp7plRPS_ea-WxrT